orchagent/intfsorch.cpp

```diff
@@ -159,6 +160,21 @@ string IntfsOrch::getRouterIntfsAlias(const IpAddress &ip, const string &vrf_nam
     return string();
 }
 
+bool IntfsOrch::isInbandIntfInMgmtVrf(const string& alias)
+{
+    if (m_syncdIntfses.find(alias) == m_syncdIntfses.end())
+        return false;
+
+    string vrf_name = "";
+    vrf_name = m_syncdIntfses[alias].vrf_name;
```

Comment:
I would suggest, a simple approach would be to use the `getVRFname `from `vrf_id `here. So you dont have to add this vrf_name to the structure.

Reply:
done